pkg/agent/variables.go

```diff
@@ -107,8 +107,11 @@ func getWindowsCustomDataVariables(config *datamodel.NodeBootstrappingConfigurat
 		"windowsPauseImageURL":                 cs.Properties.WindowsProfile.WindowsPauseImageURL,
 		"alwaysPullWindowsPauseImage":          strconv.FormatBool(cs.Properties.WindowsProfile.IsAlwaysPullWindowsPauseImage()),
 		"windowsCalicoPackageURL":              cs.Properties.WindowsProfile.WindowsCalicoPackageURL,
+		"configGPUDriverIfNeeded":              config.ConfigGPUDriverIfNeeded,
```

Comment:
Is this a property in agent pool?

Reply:
It is set by using agentPool.GPUProfile.InstallGPUDriver

You can check line 111 on resourceprovider/server/microsoft.com/common/agentbaker/nodebootstrappingconfig.go